### ISSUE TITLE
Click handler on an image

### ISSUE THREAD

**Issue by oliverjam** [FAC-Sixteen/farage-barrage#22]
https://github.com/FAC-Sixteen/farage-barrage/blob/12c80ef43c0f02d54d56a29b554167866a617b5b/src/Player/Player.js#L31-L37

Generally if you're putting a click handler on something it should be a button. That way assistive tech knows it's interactive and you get all the keyboard focus/handling (enter/spacebar to click) for free
